transcend/types/data_silo.go

```diff
@@ -131,10 +159,32 @@ func ReadDataSiloIntoState(d *schema.ResourceData, silo DataSilo) {
 	// d.Set("team_names", ...)
 	// d.Set("depended_on_data_silo_ids", ...)
 	// d.Set("data_subject_block_list_ids", ...)
-	// d.Set("headers", ...)
 	// d.Set("api_key_id", ...)
 }
 
+func ToPlaintextContextList(plaintextContexts []interface{}) []PlaintextContextInput {
+	vals := make([]PlaintextContextInput, len(plaintextContexts))
+	for i, rawContext := range plaintextContexts {
+		context := rawContext.(map[string]interface{})
```

Comment:
Github is making this hard to find, but where is `rawContext` sourced from? And are we confident that this type cast will always work? I thought the preferred way of doing something like this is to do something like:

```go
context, ok := ctx.(map[string]interface{})
if !ok {
	return err
}
```

Doesn't matter if we're confident that the incoming context will map properly